subprojects/detekt.yml

```diff
@@ -675,8 +675,10 @@ style:
     RedundantHigherOrderMapUsage:
         active: false
     # https://detekt.github.io/detekt/style.html#redundantvisibilitymodifierrule
+    # todo don't know about kotlin strict mode
+    # possibly fixed in 1.15 https://github.com/detekt/detekt/issues/3125
```

Comment:
Can we just update to 1.15?

Reply:
yep, i thought it would be better in separate pr but whatever

Reply:
fix in 1.15 https://github.com/detekt/detekt/issues/3125 only works per module, not in our detektAll task
because of how strict api detection works